REQUIRE

```diff
@@ -1,8 +1,8 @@
-julia 0.4
+julia 0.5
 Cairo
 Colors
-IniFile
-Tk
-Compat 0.7.16
+IniFile 0.3.1
+Tk 0.5.1
```

Comment:
earlier versions would work on 0.5, right? the min versions should be the widest over the supported julia versions - the broken deps' earlier versions could be julia upper bounded for 0.6 if necessary